Sources/LSPTestSupport/TestJSONRPCConnection.swift

```diff
@@ -189,7 +189,7 @@ private let testMessageRegistry = MessageRegistry(
   notifications: [EchoNotification.self]
 )
 
-#if swift(<5.11)
+#if compiler(<5.11)
```

Comment:
`<=5.10` might make more sense?

Reply:
We can only check `>=` or `<` so that’s not possible.